src/App.js

```diff
@@ -26,13 +26,22 @@ class App extends React.Component {
 
     // TODO: add the new Todo to the list in state
     // make sure not to mutate the existing array!
+    const newTodos = [...this.state.todos, newTodo]
```

Comment:
This works great! The spread operator creates a new array, so you're not mutating state.